### PR TITLE
Add whats new strings 8.3

### PR DESCRIPTION
**Pull request by igortomko**

### CODE REVIEW

File: strings.txt [hunk 1]
```diff
@@ -31604,3 +31604,27 @@ fa = دوباره نپرس
...
+  [whats_new_popularity_label_title]
+    en = 
+    ru = Популярное поблизости
+
+  [whats_new_popularity_label_message]
+    en = 
+    ru = Раздел «Чем заняться» поможет найти самые популярные места в городе.
+
+  [whats_new_hot_offers_title]
+    en =
```
Comment (therearesomewhocallmetim) [inline strings.txt]: Attractive offers from hotels

File: strings.txt [hunk 2]
```diff
@@ -31604,3 +31604,27 @@ fa = دوباره نپرس
...
+  [whats_new_popularity_label_message]
+    en = 
+    ru = Раздел «Чем заняться» поможет найти самые популярные места в городе.
+
+  [whats_new_hot_offers_title]
+    en = 
+    ru = Выгодные предложения отелей
+
+  [whats_new_hot_offers_message]
+    en =
```
Comment (therearesomewhocallmetim) [inline strings.txt]: You can search for them using the Hotels category in the search. The offers will be marked by the percentage symbol in the map and in the search results.

File: strings.txt [hunk 3]
```diff
@@ -31604,3 +31604,27 @@ fa = دوباره نپرس
   [subtitle_error_downloading_bookmarks]
     en = Bookmarks were not downloaded
     ru = Метки не были загружены
+
+  [whats_new_bookmarks_catalog_title]
+    en =
```
[the remark this answers is not in the document]
Comment (Zverik) [reply strings.txt]: Может, "MAPS.ME has got a route catalogue"? Мне ещё не очень нравится слово catalogue: может, заменить его везде на library или directory?

File: strings.txt (hunk 2 above)
Comment (Zverik) [reply strings.txt]: Лучше так: "You can find these from the Hotels category in the search panel. The offers are marked with a percent symbol on the map and in the search results".

File: strings.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Zverik) [reply strings.txt]: Артём сказал, что можно поменять на directory. Можешь сделать «MAPS.ME has got a route directory»?

**Pull request merged**